Adjust platform API test for SN2201

stephenxs opened the pull request:
1. RJ45 ports are also implemented by SFP API, which affects platform API `get_num_sfps` test.
   Test case needs to be adjusted accordingly
2. Supply default watchdog time definition for SN2201

Signed-off-by: Stephen Sun <stephens@nvidia.com>

<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/Azure/SONiC/blob/gh-pages/CONTRIBUTING.md

Please provide following information to help code review process a bit easier:
-->
### Description of PR
<!--
- Please include a summary of the change and which issue is fixed.
- Please also include relevant motivation and context. Where should reviewer start? background context?
- List any dependencies that are required for this change.
-->

Summary:
Fixes # (issue)

### Type of change

<!--
- Fill x for your type of change.
- e.g.
- [x] Bug fix
-->

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)


### Back port request
- [ ] 201911
- [ ] 202012

### Approach
#### What is the motivation for this PR?

#### How did you do it?

#### How did you verify/test it?

#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation
<!--
(If it's a new feature, new test case)
Did you update documentation/Wiki relevant to your implementation?
Link to the wiki page?
-->

keboliu commented on tests/platform/test_sfp.py:
```diff
@@ -153,7 +215,14 @@ def test_check_sfp_low_power_mode(testbed_devices, conn_graph_facts):
         assert parsed_lpmode[intf].lower() == "on" or parsed_lpmode[intf].lower() == "off", "Unexpected SFP lpmode"
 
     logging.info("Try to change SFP lpmode")
+    changed_physical_ports = set()
```
"changed" to "tested"?

stephenxs replied:
fixed

keboliu commented on tests/platform/test_sfp.py:
```diff
@@ -153,7 +215,14 @@ def test_check_sfp_low_power_mode(testbed_devices, conn_graph_facts):
     for intf in conn_graph_facts["device_conn"]:
+        phy_intf = portmap[intf][0]
+        if phy_intf in changed_physical_ports:
+            logging.info("skip restore {} to avoid repeating operating physical interface {}".format(intf, phy_intf))
```
"skip tested SFP"?

stephenxs replied:
fixed

stephenxs commented:
Community [PR #1338](https://github.com/Azure/sonic-mgmt/pull/1338) created. Close this one.

stephenxs committed:
Adjust platform API test for SN2201

On SN2201, RJ45 ports are also implemented by SFP API.
Test case needs to be adjusted accordingly

Signed-off-by: Stephen Sun <stephens@nvidia.com>

stephenxs committed:
Add default watchdog time for Alligator

Signed-off-by: Stephen Sun <stephens@nvidia.com>
